articles/nmap-network-scanner/index.md

```diff
@@ -0,0 +1,301 @@
+#### Example 1.1 simple port scan ```scanme.nmap.org```
+```console
+
+jumamiller@janabi$ nmap scanme.nmap.org
+Starting Nmap 7.91 ( https://nmap.org ) at 2021-01-12 08:36 EAT
+Nmap scan report for scanme.nmap.org (45.33.32.156)
+Host is up (0.40s latency).
+Other addresses for scanme.nmap.org (not scanned): 2600:3c01::f03c:91ff:fe18:bb2f
+Not shown: 866 closed ports
+PORT      STATE    SERVICE
+1/tcp     open     tcpmux
+7/tcp     open     echo
+22/tcp    open     ssh
+42/tcp    open     nameserver
+53/tcp    filtered domain
+80/tcp    open     http
+146/tcp   open     iso-tp0
+211/tcp   open     914c-g
+306/tcp   open     unknown
+366/tcp   open     odmr
+407/tcp   open     timbuktu
+417/tcp   open     onmux
+481/tcp   open     dvs
+541/tcp   open     uucp-rlogin
+563/tcp   open     snews
+593/tcp   open     http-rpc-epmap
+668/tcp   open     mecomm
+
+```
+
+### Getting Started
+Nmap was initially developed to run only on Unix based systems. Windows version was released in 2000, but with a few limitations including:-
+* Microsoft dropped the support for raw```TCP/IP socket```, it has proven hard to scan various VPN clients since Nmap supports ```ethernet``` interfaces only.
+* Windows Networking APIs are known to have several deficiencies hence not efficient enough.  
+* Scanning your machine from itself has proven hard to execute, hence impossible.
+
+### Nmap Installation
+
+In this tutorial, we will be using a Linux based system, 
+
+```console
+
+jumamiller@janabi$ lsb_release -a
+No LSB modules are available.
+Distributor ID: Ubuntu
+Description:    Ubuntu 20.04.1 LTS
+Release:    20.04
+Codename:   focal
+
+```
+But you're free to follow along with [MacOs](https://nmap.org/download.html) and [Windows](https://geekflare.com/nmap-on-windows/).
+
+To install Nmap in Ubuntu:-  
+ 
+ #### Requirements:-
+ * Sudo privileges- you should have the installation rights on the system.
+ * An access to ubuntu terminal (Ctrl + T).  
+ * An advanced package tool (apt) package manager or snap store.
+  
+  #### Step 1: Updating Package List
+  Before installing Nmap in your system, ensure that your software packages are up-to-date by running the following command:
+
+  ```console
+  sudo apt-get update
+   ```
+```console
+jumamiller@OpijaKaeli:~$ sudo apt-get update
+
+[sudo] password for jumamiller: 
+Hit:1 http://ke.archive.ubuntu.com/ubuntu focal InRelease                      
+Hit:2 http://dl.google.com/linux/chrome/deb stable InRelease                   
+Hit:3 http://ke.archive.ubuntu.com/ubuntu focal-updates InRelease              
+Hit:4 http://ke.archive.ubuntu.com/ubuntu focal-backports InRelease            
+Hit:5 http://ppa.launchpad.net/micahflee/ppa/ubuntu focal InRelease            
+Hit:6 https://download.docker.com/linux/ubuntu focal InRelease                 
+Hit:7 http://security.ubuntu.com/ubuntu focal-security InRelease
+Reading package lists... Done
+jumamiller@OpijaKaeli:~$ 
+
+```
+
+#### Step 2: Install Network Mapper(Nmap)
```

Comment:
Ensure spaces before parentheses. Install Network Mapper (Nmap). I observed it throughout your article. Change it wherever applicable.